emukit/model_wrappers/gpy_model_wrappers.py

```diff
@@ -108,9 +108,9 @@ def predict_covariance(self, X: np.ndarray, with_noise: bool=True) -> np.ndarray
     def get_covariance_between_points(self, X1: np.ndarray, X2: np.ndarray) -> np.ndarray:
         """
         Calculate posterior covariance between two points
-        :param X1: An array of shape n_points1 x n_dimensions. It is the first argument of the
+        :param X1: An array of shape n_points1 x n_dimensions that contains a data single point. It is the first argument of the
                    posterior covariance function
-        :param X2: An array of shape n_points2 x n_dimensions. This is the second
+        :param X2: An array of shape n_points2 x n_dimensions that may contain multiple data points. This is the second
```

Comment:
Both X1, X2 can contain multiple data points. `that may contain multiple data points` should either appear in the doc-string for both, or none, but I feel like it's somewhat implied by the shape type-hint `n_points2 x n_dimensions`

Reply:
Done (and cleaned up a bit more too).